XamarinCommunityToolkitSample/Pages/Behaviors/MaskedBehaviorPage.xaml

```diff
@@ -0,0 +1,88 @@
+﻿<?xml version="1.0" encoding="UTF-8" ?>
+<pages:BasePage x:Class="Microsoft.Toolkit.Xamarin.Forms.Sample.Pages.Behaviors.MaskedBehaviorPage"
+                xmlns="http://xamarin.com/schemas/2014/forms"
+                xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+                xmlns:behaviors="clr-namespace:Microsoft.Toolkit.Xamarin.Forms.Behaviors;assembly=Microsoft.Toolkit.Xamarin.Forms"
+                xmlns:pages="clr-namespace:Microsoft.Toolkit.Xamarin.Forms.Sample.Pages"
+                xmlns:resources="clr-namespace:Microsoft.Toolkit.Xamarin.Forms.Sample.Resx;assembly=Microsoft.Toolkit.Xamarin.Forms.Sample">
+    <ScrollView>
+        <StackLayout Padding="{StaticResource ContentPadding}" Spacing="5">
+
+            <Label Padding="10,10,10,50" Text="{x:Static resources:AppResources.MaskedDescription}" />
+
+            <Editor>
+                <Editor.Behaviors>
+                    <behaviors:MaskedBehavior Mask="XX-XX-XX" />
+                </Editor.Behaviors>
+            </Editor>
+            <Label Padding="10,0,10,50"
+                   FontSize="Small"
+                   Text="Editor test, Sort code 'XX-XX-XX'" />
+
+            <Entry Keyboard="Numeric">
+                <Entry.Behaviors>
+                    <behaviors:MaskedBehavior Mask="XX-XX-XX" />
+                </Entry.Behaviors>
+            </Entry>
+            <Label Padding="10,0,10,50"
+                   FontSize="Small"
+                   Text="e.g. Sort code 'XX-XX-XX'" />
```

Comment:
Would you be able to put the texts in the resource file that's there so it will work for the multi-language things?

Reply:
@jfversluis  Ye sure is there any naming conversion for the resources?

Reply:
@Char0394 ?